### PR TITLE
Fgoschtest/test1

### PR DESCRIPTION
**Pull request by Fgosch**

### CODE REVIEW

File: sube/Podfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'sube' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
```
Comment (chuckSaldana) [inline sube/Podfile:5]: Is this comment really necessary?

File: demo.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Ich habe hunger
```
Comment (chuckSaldana) [inline demo.txt:1]: This doesn't make sense. Change it please.

Comment (Fgosch) [reply demo.txt:1]: ok U,U

**Comment by chuckSaldana** [comment]
+1

**Comment by chuckSaldana** [comment]
+1

**Pull request merged**